docs/source/model_design/vivarium_model_components/causes/index.rst

```diff
@@ -865,5 +866,11 @@ Incidence- vs. Prevalence-Based YLDs
      - YLD and YLL measurements are measured using different methods; For diseases with a short duration, YLDs may be underestimated.
 
 
+YLDs in Vivarium 
+^^^^^^^^^^^^^^^^
+DWs and YLDs in Vivarium apply to the duration of the condition. Remember, YLDs are a product of the DW associated with the condition and the time between onset and survival. To illustrate an example, let's say an individual contracted the flu with a disability weight of 0.05 and recovered after 1 week. To calculate the YLDs here, we would multiply 0.05 by 1/52 (0.019).
+
+Another important note is that DWs and YLDs must always be less than 1, because '1' is essentially death. For this reason, if an individual experiences multiple conditions at once, the total amount of YLDs is calculated by multiplying the YLDs of each individual condition together. For instance, let's say an individual has lived with three health conditions, with respective YLDs of 0.3, 0.4, and 0.5. The total YLDs here would be (0.3) * (0.4) * (0.5). 
```

Comment:
```suggestion
Another important note is that DWs and YLDs must always be less than 1, because '1' is essentially death. For this reason, if an individual experiences multiple conditions at once, the total amount of YLDs is calculated by multiplying the YLDs of each individual condition together. For instance, let's say an individual has lived with three health conditions, with respective YLDs of 0.3, 0.4, and 0.5. The total YLDs here would be :code:`1 - ((1-0.3) * (1-0.4) * (1-0.5))`. 
```